### PR TITLE
Implementing multi-row insert for liftQuery(...).foreach

### PR DESCRIPTION
**Pull request by deusaquilus**
Implementing multi-insertion capabilities for most contexts via the VALUES (clauseA), (clauseB), (clauseC), etc... construct. The performance benefits of this when working of a network with any kind of real latency are significant e.g. insertion of 1 million rows goes down from 6.6 minutes to 10 seconds!

### CODE REVIEW

File: quill-jdbc/src/main/scala/io/getquill/context/jdbc/JdbcContextVerbExecute.scala [hunk 1]
```diff
@@ -78,7 +78,7 @@ trait JdbcContextVerbExecute[+Dialect <: SqlIdiom, +Naming <: NamingStrategy] ex
       groups.flatMap {
         case BatchGroup(sql, prepare) =>
           val ps = conn.prepareStatement(sql)
-          logger.underlying.debug("Batch: {}", sql)
+          //logger.underlying.debug("Batch: {}", sql.take(200) + (if (sql.length > 200) "..." else ""))
```
Comment (analytically) [inline quill-jdbc/src/main/scala/io/getquill/context/jdbc/JdbcContextVerbExecute.scala:81]: unintentional?

Comment (deusaquilus) [reply quill-jdbc/src/main/scala/io/getquill/context/jdbc/JdbcContextVerbExecute.scala:81]: Whoops. I’ll fix that

File: scripts/increase_postgres_latency.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Make the Postgres DB image have 100ms latency for all requests.
```
Comment (analytically) [inline scripts/increase_postgres_latency.sh:3]: 50ms

**Comment by jilen** [comment]
Are all variables sanitized ?  I cannot find anything related. 

 For jdbc, it is safe to execute via `addBatch`. Not sure about other contexts.

**Comment by deusaquilus** [comment]
@jilen 
> Are all variables sanitized ? I cannot find anything related.
Have a look at [Particularize](https://github.com/zio/zio-protoquill/blob/master/quill-sql/src/main/scala/io/getquill/context/Particularize.scala). It's like an advanced version of ReifyLiftings.

> For jdbc, it is safe to execute via `addBatch`. Not sure about other contexts.
I'm only supporting this functionality for JDBC contexts for now. This will need to be tested separately for Jasync and others.

**Pull request merged**